### PR TITLE
feature: replace enter to new dashboard in beta

### PR DESCRIPTION
**Pull request by nuanyang233**
closes #

### CODE REVIEW

**Comment by septs** [comment]
CC @DimensionDev/qa plz test the pr.

**Comment by septs** [comment]
@nuanyang233 plz rebase develop latest commit

File: packages/maskbook/src/extension/options-page/Welcome/getWelcomePageURL.tsx [hunk 1]
```diff
@@ -7,6 +7,8 @@ export function getWelcomePageURL() {
         return browser.runtime.getURL(`index.html#${DashboardRoute.Setup}`)
     } else if (Flags.has_no_browser_tab_ui) {
         return browser.runtime.getURL(`index.html#${DashboardRoute.Nav}`)
+    } else if (Flags.v2_enabled) {
+        return browser.runtime.getURL(`next.html#/Welcome`)
```
Comment (Jack-Works) [inline packages/maskbook/src/extension/options-page/Welcome/getWelcomePageURL.tsx:11]: No, it's not ready I guess

File: packages/maskbook/src/extension/browser-action/UI.tsx [hunk 2]
```diff
@@ -82,7 +82,7 @@ function BrowserActionUI() {
                 active: true,
                 url: browser.runtime.getURL('/debug.html'),
             })
-        } else if (process.env.NODE_ENV === 'development' && event.ctrlKey) {
+        } else if (Flags.v2_enabled) {
```
Comment (Jack-Works) [inline packages/maskbook/src/extension/browser-action/UI.tsx:85]: No, it's not ready I guess

File: packages/maskbook/src/extension/options-page/Welcome/getWelcomePageURL.tsx (hunk 1 above)
Comment (nuanyang233) [reply packages/maskbook/src/extension/options-page/Welcome/getWelcomePageURL.tsx:11]: Just merge in `develop-2.0` for qa test

**Comment by wenluomask** [comment]
Checked.

**Pull request merged**